sdks/java/core/src/main/java/org/apache/beam/sdk/metrics/Lineage.java

```diff
@@ -23,11 +23,8 @@
 public class Lineage {
 
   public static final String LINEAGE_NAMESPACE = "lineage";
-  public static final String SOURCE_METRIC_NAME = "sources";
```

Comment:
this is breaking change but Lineage is just introduced for one release and only used internally. I would suggest change the constant String to a enum so we can have a query method guarantees the metric name is among defined ones